go.mod

```diff
@@ -4,62 +4,54 @@ go 1.17
 
 require (
 	github.com/onsi/ginkgo v1.16.5
```

Comment:
If nothing uses ginkgo v1, should this still be here?

Reply:
No 😞 I'll dig into it